deploy/helm/demarkus-broker/templates/secret-issuances.yaml

```
@@ -0,0 +1,36 @@
+     on the LIVE Secret prevents that. Net effect: chart updates never
+     touch the broker's runtime state.
+
+  `lookup` returns nil during `helm template` (no cluster context), so
+  offline renders always emit the empty Secret. helm-unittest assertions
+  see the empty Secret and can verify the resource-policy annotation.
+*/}}
+{{- if not (lookup "v1" "Secret" .Release.Namespace (include "demarkus-broker.issuancesSecretName" .)) }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "demarkus-broker.issuancesSecretName" . }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)